ConsoleApplication1/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
```

Comment:
Please remove or rename "ConsoleApplication1" or remove if it's in by mistake.

Please try and work neatly